jaws-journey-deploy/orb.yml

```diff
@@ -49,6 +53,11 @@ aliases:
     var_file: << parameters.environment >>.tfvars
     label: << parameters.environment >>
 
+  slack-notification: &slack-notification
+    event: fail
+    mentions: '@jaws_support'
```

Comment:
It is "@jaws-support"

Reply:
Should this also come from config though?

Reply:
I think OSO are using this orb for JAWS API and they might want to notify their support

Reply:
yeah its jaws-support.  Passing it into the orb, might be annoying, only option I can think of would be as a env var which is also a bit rubbish, and wouldnt help with OSO example (as they use the JAWS contexts).  Will have a think, but right now I think having it set to jaws-support regardless is fine - OSO can look in the alert channels for now